ivy/functional/ivy/layers.py

```diff
@@ -131,6 +131,13 @@ def linear(
     }
     
     """
+    if ivy.current_backend(x).backend == 'torch':
```

Comment:
We shouldn't need to do this, once the backend is set to `torch`, `ivy.linear` would automatically defer to the `torch`-backend implementation as we discussed yesterday